.github/workflows/release.yml

```diff
@@ -1,41 +1,23 @@
 name: Release
 
 on:
-  push:
-    branches:
-      - main
-      - rc
-
+  release:
+    types: [published]
+  workflow_dispatch:
+    description: "Manually publish release"
```

Comment:
Seems like this conflicts with `prepare-release.yml::workflow_dispatch`. I think we're good here with just `on: release: ...`